Revert part of upstream commit that added a dependency on the
nonexistent 'iconv.pc':
<https://github.com/libarchive/libarchive/commit/a83f3d327aefe8d628230f326621b014d82f55b4>
--- build/pkgconfig/libarchive.pc.in.orig	2022-12-10 00:38:31.000000000 +1100
+++ build/pkgconfig/libarchive.pc.in	2022-12-17 11:28:03.000000000 +1100
@@ -10,4 +10,3 @@
 Cflags.private: -DLIBARCHIVE_STATIC
 Libs: -L${libdir} -larchive
 Libs.private: @LIBS@
-Requires.private: @LIBSREQUIRED@
